pydicer/convert/HN_137_RTstruct_to_Niffti_all - PET.py

```diff
@@ -0,0 +1,109 @@
+###################### Read Me #######################

```

Comment:
The code in this file seems like it should be integrated into the pydicer pipeline. We have a module for fetching DICOM files, preprocessing and linking them, and then finally converting them. This code could provide the functionality to handle PT data in that pipeline.